src/main/java/core/basesyntax/FileWork.java

```diff
@@ -9,6 +15,16 @@
  */
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        Path pathToFile = Paths.get(fileName);
+        if (Files.exists(pathToFile) & Files.isReadable(pathToFile)) {
+            try {
+                return Arrays.stream(Files.readString(pathToFile)
+                        .split("\\W")).map(String::toLowerCase)
+                        .filter(x -> x.startsWith("w")).sorted().toArray(String[]::new);
```

Comment:
```suggestion
                return Arrays.stream(Files.readString(pathToFile)
                        .split("\\W"))
                        .map(String::toLowerCase)
                        .filter(x -> x.startsWith("w"))
                        .sorted()
                        .toArray(String[]::new);
```
Let's write each stream method in a new line. This way code will be more readable